### DISCUSSION TITLE
Bitbucket Compatibility?

### DISCUSSION THREAD

**Discussion by cluelessone** [jv-k/ver-bump#31] [Q&A] [answered]
This looks like a great tool. Any chance it is compatible with Bitbucket? I tried poking around but couldn't find where Github is baked into it.
Thanks in advance!

**Comment by jv-k** [comment, marked as answer]
Hi there,

`ver-bump` uses the git command in several places. See [https://github.com/jv-k/ver-bump/blob/main/lib/helpers.sh]()

I used to use BitBucket myself. Your local repo that is hosted on there would be connected to your BitBucket remote repository with `git remote add origin <server_URL>` as if it was hosted on GitHub — it's already compatible. And therefore `ver-bump` will happily work with Bitbucket.

Thanks for your interest.
